sdk/search/azure-search/README.md

```diff
@@ -0,0 +1,252 @@
+# Azure cognitive search client library for Java
+
+This is the Java client library for [Azure Cognitive Search](https://docs.microsoft.com/en-us/rest/api/searchservice/).
+Azure Cognitive Search is a fully managed cloud search service that provides a rich search experience to custom applications.
+This library provides an easy (native) way for a Java developer to interact with the service to accomplish tasks like: create and manage indexes, load data, implement search features, execute queries, and handle results.
+
+[Source code][source_code] | [Package (Maven)][package] | [API reference documentation][api_documentation]
+| [Product documentation][search_docs] | [Samples][samples]
+
+## Getting started
+
+### Prerequisites
+
+-   Java Development Kit (JDK) with version 8 or above
+-   [Azure subscription][azure_subscription]
+-   An instance of [Cognitive search service][search]
+
+### Adding the package to your product
+
+[//]: # {x-version-update-start;com.azure:azure-search;current}
+
+```xml
+<dependency>
+    <groupId>com.azure</groupId>
+    <artifactId>azure-search</artifactId>
+    <version>1.0.0-preview.2</version>
+</dependency>
+```
+
+[//]: # {x-version-update-end}
+
+### Default HTTP Client
```

Comment:
The Default HTTP Client, Alternate HTTP client, Configuring HTTP Clients,  Default SSL library sections need to be updated.
Example: https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/appconfiguration/azure-data-appconfiguration/README.md
@mssfang

Reply:
Correct.  Please remove these `Default HTTP Client`, `Alternate HTTP client`, `Configuring HTTP Clients` and `Default SSL library` sections here.  And copy and paste `Default HTTP Client` and `Default SSL library` sections from AzConfig README to Troubleshootings sections.

Reply:
fixed

Reply:
@navalev  this issue is still open.

Reply:
@mssfang Default HTTP Client, Alternate HTTP client, Configuring HTTP Clients and Default SSL library sections sections where removed. Default HTTP Client and Default SSL library where copied from azConfig to the troubleshooting section. What other item is missing?

Reply:
No other things are missing. Yesterday I saw there are no changes to these sections. It looks good now.